src/perf_counters.cc

```diff
@@ -117,7 +117,7 @@ bool PerfCounters::Initialize() { return false; }
 PerfCounters PerfCounters::Create(
     const std::vector<std::string>& counter_names) {
   if (!counter_names.empty()) {
-    GetErrorLogInstance() << "Performance counters not supported.";
+    VLOG(3) << "Performance counters not supported.\n";
```

Comment:
VLOG(1) or VLOG(2) for this?

Reply:
Okay. I thought 1 was default (it is 0). I can change all the above levels to 1.
The long term solution is to validate the command-line args, and print the diagnostics before even starting the first benchmark.
But this will take a lot of code changes. The current counter-error-logging code is tightly coupled with counter-reading code.

Reply:
The current error logging code assumes we're in PerfCounters::Create because we actually want to create counters. I think the fix is to catch outside calling this that either they aren't supported, or the vector is empty, and not call it.